### DISCUSSION TITLE
Using pseudo BNF codes with the codelist_from_csv function

### DISCUSSION THREAD

**Discussion by SRW612** [opensafely/documentation#343] [Codelists]
Just a very small question about the codelist_from_csv function: if I am using this to import pseudo BNF codes, what do I put for the 'system=' option? The documentation mentions system="snomed" and system="ctv3" but I can't find the equivalent for pseudo BNF anywhere.

**Comment by brianmackenna** [comment]
On OpenCodelists we make `pseudoBNF` available as there is a good human structured hierarchy which makes it easy to build codelists. However   you'll need to use a NHS dictionary of medicines and devices `dm+d` codelists to query using `with_these_medications`.

`dm+d` is effectively a local UK snomed extension and we have plenty available on opencodelists. If you have made a new pseudoBNF list we can help convert it to dm+d if you can share the url?

**Reply by SRW612** [reply]
Thanks for the explanation of dm+d. I just tried downloading the dm+d version of my codelist, then using that to create a new snomed-based codelist, although the codes are mainly unknown and I am stopped at the csv upload stage.

I have already created a pseudo BNF codelist, if you are able to help convert it: 
https://www.opencodelists.org/codelist/user/S_Walter/antibiotics/4ccf1e1e/
